docs/src/extend/shareable-configs.md

```diff
@@ -90,13 +94,15 @@ You can override settings from the shareable config by adding them directly into
 
 ```js
 // eslint.config.js
+import { defineConfig } from "eslint/config";
 import myconfig from "eslint-config-myconfig";
 
 export default [
```

Comment:
```suggestion
export default defineConfig([
```

plus the corresponding closing `)` on line 110 below (GitHub doesn't let me leave a suggestion there).